cleanlab_studio/studio/studio.py

```diff
@@ -361,3 +362,11 @@ def download_embeddings(
         Downloads embeddings for a cleanset
         """
         return np.asarray(api.download_array(self._api_key, cleanset_id, "embeddings"))
+
+    def TLM(self, *, quality_preset: trustworthy_llm.QualityPreset = "low") -> trustworthy_llm.TLM:
+        """Gets trustworthy language model (TLM) object to prompt.
+
+        Returns:
+            TLM: the trustworthy language model object
+        """
```

Comment:
Add a runtime check here to ensure that `quality_preset` is reasonable, otherwise raise ValueError.

Right now, `studio.TLM(quality_preset="foo").prompt("bar")` gives an internal server error.